File: iOS/MyCloud/MyCloud/MenmoryCache/Memory.m
//
//  Menmory.m
//  MyCloud
//
//  Created by clare chen on 9/6/16.
//  Copyright © 2016 clare chen. All rights reserved.
//

#import "Memory.h"

static User* user;
static NSString *token;

@implementation Memory

+(User*)getUser{
    return user;
}

+(void)setUser:(User*) u{
    user = u;
}

+(NSString*)getToken{
    return token;
}

+(void)setToken:(NSString*)t{
    token = t;
}


@end

File: Android/mycloud/app/src/main/java/com/mycloud/memory/Memory.java
package com.mycloud.memory;

import com.mycloud.pojo.User;

/**
 * Created by clarkchen on 08/12/2016.
 */

public class Memory {

    private static User user;
    private static String token;

    public static User getUser() {
        return user;
    }

    public static void setUser(User user) {
        Memory.user = user;
    }

    public static String getToken() {
        return token;
    }

    public static void setToken(String token) {
        Memory.token = token;
    }




}
